net/pfSense-pkg-cellular/files/usr/local/bin/cellular_dev.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/local/bin/python2.7
+#
+# /usr/local/bin/cellular_dev.py
+#
+# part of pfSense (https://www.pfsense.org)
+# Copyright (C) 2017 Voleatech GmbH
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+Helper to mount cellular modem at fixed dev point
+"""
+
+import sys
+import os
+import commands
+
+import argparse
+
+data_file = "cuaZ99.0"
+data_path = "/dev/" + data_file
+mgm_file = "cuaZ99.1"
+mgmt_path = "/dev/" + mgm_file
+
+def remove_link():
+
+    if os.path.lexists(data_path):
+        os.system("rm " + data_path)
+
+    if os.path.lexists(mgmt_path):
+        os.system("rm " + mgmt_path)
+
+    #Remove locks
+    data_lock = "/var/spool/lock/LCK.." + data_file
+    if os.path.exists(data_lock):
+        os.system("rm " + data_lock)
+    
+    mgmt_lock = "/var/spool/lock/LCK.." + mgm_file
+    if os.path.exists(mgmt_lock):
+        os.system("rm " + mgmt_lock)
+
+# Argument parser
+parser = argparse.ArgumentParser(description = "Interface for Cellular Dev Point")
+
+parser.add_argument("-a", "--add",
+        action="store_true",
+        dest="add",
+        help="add modem")
+parser.add_argument("-r", "--remove",
+        action="store_true",
+        dest="remove",
+        help="remove modem")
+parser.add_argument("-d", "--device",
+        dest="device",
+        help="device name")
+
+
+# parse commandline args
+args = parser.parse_args()
+args.silent = False
+
+if args.add:
+    if args.device:
+
+        #Make sure the links are gone
+        remove_link()
+
+        #Before we start kill 3gstat as it blocks us
+        os.system("/bin/ps auxww | /usr/bin/grep \"[3]gstats\" | /usr/bin/awk '{print $2}' | /usr/bin/xargs kill")
```

Comment:
Using pkill here would be better, everything can be done with that one command, for example:

`/bin/pkill -fx '.*[3]gstats.*'`